aws/v12.1.3/README.md

```diff
@@ -0,0 +1,20 @@
+# :zap: Giant Swarm Release v12.1.3 for AWS :zap:
+
+This release provides a new cluster-operator which fixes preventing release upgrade when reference id does not align with the G8sControlPlane id or MachineDeployment id.
```

Comment:
Maybe it could be rephrased. Something like:
```suggestion
This release provides a new cluster-operator which fixes the issue that prevented release upgrade when reference id does not align with the G8sControlPlane id or MachineDeployment id.
```